ibm/conns/config.go

```diff
@@ -2381,6 +2392,34 @@ func (c *Config) ClientSession() (interface{}, error) {
 		// })
 	}
 
+	// Construct an instance of the 'Configuration Aggregator' service.
+	if session.configurationAggregatorClientErr == nil {
+		// Construct the service options.
+		authenticator := &core.IamAuthenticator{
+			ApiKey: "<the_apikey>",
```

Comment:
use `Authenticator` instead of `ApiKey`. Check other client sessions for reference in the same file.